src/satosa/frontends/openid_connect.py

```diff
@@ -460,7 +468,7 @@ def _create_provider(
     return provider
 
 
-def _init_authorization_state(provider_config, db_uri, sub_hash_salt):
+def _init_authorization_state(provider_config, db_uri, sub_hash_salt, mirror_public):
```

Comment:
even though this is a "private" function, it is better to make the new param optional as it defaults to `False` anyway

```suggestion
def _init_authorization_state(provider_config, db_uri, sub_hash_salt, mirror_public=False):
```

Reply:
Thanks, I've updated the commit. Additionally, I've spotted that there was a lingering assert left in the test code (as a result of a copy-paste).